lib/plugins.js

```diff
@@ -570,7 +570,7 @@ function plugins (settings, deviceId) {
       }
       : null
 
-    const cassette1Alert = cashOutEnabled && device.cassette1 < notifications.fiatBalanceCassette1
+    const cassette1Alert = cashOutEnabled && (device.cassette1 / 500) < notifications.fillingPercentageCassette1
```

Comment:
wondering if there's a variable somewhere in the repo for the cash cassette max capacity, instead of just hard coding 500.